{{cookiecutter.github_repository}}/{{cookiecutter.main_module}}/urls.py

```diff
@@ -11,15 +11,21 @@
 from django.conf.urls import include, url
 from django.conf.urls.static import static
 from django.contrib import admin
+from django.views.defaults import (bad_request,
+                                   page_not_found,
+                                   permission_denied)
+
+# {{ cookiecutter.project_name }} Stuff
+from {{ cookiecutter.main_module }}.base.views import (server_error, root_txt_files)
```

Comment:
Let's use `from {{ cookiecutter.main_module }}.base import views as base_views` and `from django.views import defaults as dj_default_views`.